.gitignore

```diff
@@ -91,3 +91,7 @@ bundler.d/*
 # yardoc
 /doc/
 .yardoc
+
+# mac hidden dirs
```

Comment:
This feels like something that should be in your `~/.gitignore`

Reply:
Yes it was but git couldn't ignore it. Since the last commit, I suppressed their creation using `defaults write com.apple.desktopservices DSDontWriteNetworkStores true`